spring-grpc-spring-boot-autoconfigure/pom.xml

```diff
@@ -20,79 +20,54 @@
 		<developerConnection>git@github.com:spring-projects-experimental/spring-grpc.git</developerConnection>
 	</scm>
 
-	<dependencies>
+	<dependencyManagement>
```

Comment:
Moved the dep. on SB down into autoconfigure as we will have chicken-egg once we moving into Spring Boot.